### PR TITLE
Benchmarking parallel

### LINKED ISSUE #106: setup CI to generate benchmark weights

### PR DESCRIPTION
**Pull request by ayushmishra2005**
resolves https://github.com/parallel-finance/parallel/issues/106

### CODE REVIEW

**Comment by 0x8f701** [comment]
please squash and use only one commit message @ayushmishra2005

File: bench.js [hunk 1]
```diff
@@ -153,16 +156,16 @@ var SubstrateRuntimeBenchmarkConfigs = {
...
-            '--output=./frame/{pallet_folder}/src/weights.rs',
+            '--output=./pallets/{pallet_folder}/src/weights.rs',
             '--template=./.maintain/frame-weight-template.hbs',
         ].join(' '),
     },
     "substrate": {
         title: "Benchmark Runtime Substrate Pallet",
         branchCommand: [
-            'cargo run --release',
+            '/home/alex_cj96/.cargo/bin/cargo run --release',
```
Comment (0x8f701) [inline bench.js]: can we have a config section for this?

like `CARGO_PATH`

File: bench.js [hunk 2]
```diff
@@ -61,6 +61,8 @@ var BenchConfigs = {
     }
 }
 
+
+
```
Comment (kaichaosun) [inline bench.js]: Only need one empty line.

File: bench.js [hunk 3]
```diff
@@ -140,9 +143,9 @@ var SubstrateRuntimeBenchmarkConfigs = {
     "pallet": {
         title: "Benchmark Runtime Pallet",
         branchCommand: [
-            'cargo run --release',
+            '/home/alex_cj96/.cargo/bin/cargo run --release',
```
Comment (kaichaosun) [inline bench.js]: Is this `alex..` expected?

**Review by kaichaosun — APPROVED** [review]
LGTM except a few small comments

**Comment by ayushmishra2005** [comment]
> please squash and use only one commit message @ayushmishra2005

We can click on `Squash and merge` while merging PR.

**Comment by 0x8f701** [comment]
> > please squash and use only one commit message @ayushmishra2005
> 
> We can click on `Squash and merge` while merging PR.

but the commit message will be too big, I would expect to have one because the commit message is the same

**Review by 0x8f701 — APPROVED** [review]

**Pull request merged**